src/LanguageServer/Protocol/Features/Diagnostics/EngineV2/DiagnosticIncrementalAnalyzer.InProcOrRemoteHostAnalyzerRunner.cs

```diff
@@ -195,8 +193,7 @@ public async Task<ImmutableArray<Diagnostic>> GetSourceGeneratorDiagnosticsAsync
 
             var ideOptions = ((WorkspaceAnalyzerOptions)compilationWithAnalyzers.AnalysisOptions.Options!).IdeOptions;
 
-            var analyzers = documentAnalysisScope?.Analyzers ??
-                compilationWithAnalyzers.Analyzers.Where(a => forceExecuteAllAnalyzers || !a.IsOpenFileOnly(ideOptions.SimplifierOptions));
```

Comment:
the right side of the `||` became `true`.  So this `Where` was unnecessary.  which made `forceExecuteAllAnalyzers` unnecessary.  which led to the callsite updates.